zosia_site/iam.tf

```diff
@@ -1,28 +1,32 @@
-# Required for App Engine app to access Secret Manager in runtime
+resource "google_service_account" "cloudrun_service_account" {
+  account_id   = "cloudrun-service-account"
```

Comment:
I would suggest renaming it to indicate that it is a service account for the Zosia website service (even though currently this is the only service you have in this infrastructure).

Reply:
Done